MODE: "double_AMR"
# Kinematics
TOW_CAR_LENGTH: 0.90
SHELF_LEN: 0.73
MAX_CIRCLE_RADIUS: 0.10
SEARCH_RADIUS: 0.55
# Tolerance
SHEFT_LENGTH_TOLERANCE: 0.1 # Meter
ANGLE_TOLERANCE: 6 # Degree